packages/dependency-extraction-webpack-plugin/src/index.js

```diff
@@ -17,14 +17,18 @@ function camelCaseDash( string ) {
 	return string.replace( /-([a-z])/g, ( _, letter ) => letter.toUpperCase() );
 }
 
-const wooRequestToExternal = ( request ) => {
+const wooRequestToExternal = ( request, excludedExternals ) => {
```

Comment:
`excludedExternals` is a new option value, so will need to be reflected in the README

https://github.com/woocommerce/woocommerce-admin/tree/1c16db605af1436647d990230bd146e74f582d88/packages/dependency-extraction-webpack-plugin

Reply:
Now that I think about it, this should really be upstream in Gutenberg https://github.com/WordPress/gutenberg/tree/trunk/packages/dependency-extraction-webpack-plugin if you want to make a PR there.

Reply:
Just to clarify: We should keep the changes in this PR (with README updates), but also support the `excludedExternals` param in the original plugin within Gutenberg?

Reply:
Yup, thats right

Reply:
I created a PR, I will just update this PR to follow the format I went with here: https://github.com/WordPress/gutenberg/pull/32220